docs/readmes/resources/ref_magma_metrics.md

```diff
@@ -0,0 +1,186 @@
+---
+id: ref_magma_metrics
+title: Life of a Magma Metric
+hide_title: true
+---
+
+## Life of a Magma Metric
+
+This document will cover the life of a Magma metric as it originates at the
+access gateway (AGW), goes through the orchestrator (orc8r) to the [Prometheus
+Edge Hub](https://github.com/facebookincubator/prometheus-edge-hub), and eventually gets displayed on the NMS UI.
+
+### Phase 1: Collection and Export
+
+On the AGW, metrics are collected and exported from either Python services or
+C++ services.
+
+Every **Python** service on the access gateway defines
+[Prometheus](https://prometheus.io/) Counters, Gauges, and Histograms in a
+`metrics.py` file. The `subscriberdb` service, for example, defines the
+following Prometheus metrics in its
+[metrics.py](https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/python/magma/subscriberdb/metrics.py)
+file among others:
+
+```python
+SUBSCRIBER_SYNC_SUCCESS_TOTAL = Counter(
+    'subscriber_sync_success',
+    'Total number of successful subscriber'
+    'syncs with cloud',
```

Comment:
Here and below: go ahead and just merge these two lines to avoid reader confusion, since we don't care about 80-char line lengths in documentation